R/datasets.R

```diff
@@ -125,6 +125,16 @@ NULL
 #' @docType data
 NULL
 
+#' Basque pathogen infection spatial binary classification task.
+#'
+#' Contains the task (\code{bc.task.spatial}).
+#'
+#' @name bc.task.spatial
+#' @references Iturritxa, E., Mesanza, N., & Brenning, A. (2014). Spatial
+#' analysis of the risk of major forest diseases in Monterey pine plantations.
+#' Plant Pathology, 64(4), 880-889. doi:10.1111/ppa.12328
+#'
+NULL
 
```

Comment:
Is this dataset really required for this PR? Could you use another dataset? Just trying to keep things as contained as possible.